__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const db = require("../db/connection");
 const app = require("../app");
 const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data/index");
+const { patch } = require("../app");
```

Comment:
Think this might be an auto import issue? This can cause you some headaches later